passphrase.expect

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/expect
```

Comment:
Ran a quick test locally and I don't believe `expect` is installed in this image based on the Dockerfile. You'll have to add it to the `yum install` command before this will work.